app/src/main/res/xml-eu/ext_kbd_bottom_row_ng.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In that case, let me suggest this:
Instead of altering the existing bottom rows, you may want to create an additional bottom row  with hyphen.

I would also explore the possibility to add all those bottom rows as a part of the Basque language pack.